### PR TITLE
Update Fleming content changes

### PR DESCRIPTION
**Pull request by siddique26**
Fixed #512 - Remove refs to Alpha 2

Reference doc:
https://docs.google.com/document/d/1wMwUDIWZlpYrGW_alypIhk2frod70tcZt_rlMmNbdiQ/edit

### CODE REVIEW

File: src/pages/GetInvolved/Invite/Invite.js [hunk 1]
```diff
@@ -78,19 +76,9 @@ export default class Invite extends React.Component {
...
-    }
-    this.setState({
-      showInviteOpts: state
-    })
-  }
-
   render() {
     const { content } = this.props
-    const { title, installBrowser, feedback, modal } = content
+    const { claimInvite, title, installBrowser, feedback, modal } = content;
```
Comment (joshuef) [inline src/pages/GetInvolved/Invite/Invite.js]: lets update this naming.

File: src/pages/FAQs/content.js [hunk 2]
```diff
@@ -40,31 +40,7 @@ export default {
             'Started in 2006 by Scottish engineer David Irvine, MaidSafe are the core developers of the SAFE Network.  It’s a small team comprised of: thinkers, inventors, tinkerers, PHDs, engineers and designers. Although based in Ayr, Scotland, we work remotely with talents from many different cultures and countries, reflecting the users that we serve. Despite this variety, we all share a single mission: providing security and privacy for everyone.',
             '_[Visit Maidsafe.net](https://maidsafe.net)_'
           ]
-        }
-      ]
-    },
-    {
-      title: 'Getting Started',
```
Comment (joshuef) [inline src/pages/FAQs/content.js:47]: Do we really not want this section?

File: src/pages/GetInvolved/Develop/Develop.js [hunk 3]
```diff
@@ -44,6 +45,7 @@ const Develop = props => {
             <UnderlinedLink url={coreNetwork.CTA.url}>{coreNetwork.CTA.name}</UnderlinedLink>
           </div>
         </div>
+        <Invite content={invite} />
```
Comment (joshuef) [inline src/pages/GetInvolved/Develop/Develop.js:48]: Lets rename this component too be relevant too

File: src/pages/GetInvolved/content.js [hunk 4]
```diff
@@ -59,59 +59,55 @@ export default {
...
-  //         name: 'Join Community',
-  //         url: 'https://safenetforum.org'
-  //       }
-  //     }
-  //   }
-  // },
+  invite: {
+    title: 'Get Started',
+    claimInvite: {
+      id: 'claimInvite',
```
Comment (joshuef) [inline src/pages/GetInvolved/content.js]: here too 🙇

**Review by joshuef — CHANGES_REQUESTED** [review]

File: src/pages/GetInvolved/Invite/Invite.js (hunk 1 above)
Comment (siddique26) [reply src/pages/GetInvolved/Invite/Invite.js]: okay. changes to joinCommunity

File: src/pages/FAQs/content.js (hunk 2 above)
Comment (siddique26) [reply src/pages/FAQs/content.js:47]: Sorry I'll add this

File: src/pages/GetInvolved/Develop/Develop.js (hunk 3 above)
Comment (siddique26) [reply src/pages/GetInvolved/Develop/Develop.js:48]: Don't know how and which are files to change, as style file is named invite.sass

File: src/pages/GetInvolved/content.js (hunk 4 above)
Comment (siddique26) [reply src/pages/GetInvolved/content.js]: Okay

**Review by joshuef — APPROVED** [review]

**Pull request merged**